src/features/MentorPage/MentorChips/ShowMoreChips.tsx

```diff
@@ -6,25 +6,23 @@ import ExpandIcon from '../../../static/img/icon-expand.svg';
  * Should update Skill Chips container when pressed?
  */
 
-const ShowMoreButtonHandler = () => {
-  const skillChips = document.getElementById('skill-chips');
-  const showMoreButton = document.getElementById('show-more-button');
-  if (skillChips && showMoreButton) {
-    skillChips.classList.toggle('show-more');
-    showMoreButton.innerText === 'Näytä kaikki aiheet'
-      ? (showMoreButton.innerText = 'Näytä vähemmän')
-      : (showMoreButton.innerText = 'Näytä kaikki aiheet');
-  }
+type ShowMoreSkillsProps = {
```

Comment:
Nitpicky thing: if there is only one component in file, then I would prefer the type of the props to be called just Props. It is easier to read when browsing many files then =P